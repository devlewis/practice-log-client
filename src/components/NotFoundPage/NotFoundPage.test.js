import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NotFoundPage from "./NotFoundPage";

describe(`NotFoundPage component`, () => {
  const props = {
    className: "test-class-name",
    children: <p>test children</p>,
    "data-other": "test-other-prop",
  };

  it("renders form.NotFoundPage by default", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the NotFoundPage given props", () => {
    const wrapper = shallow(<NotFoundPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
