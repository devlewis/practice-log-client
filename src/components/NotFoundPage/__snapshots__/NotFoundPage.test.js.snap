// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders form.NotFoundPage by default 1`] = `
<section
  className="NotFoundPage"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;

exports[`NotFoundPage component renders the NotFoundPage given props 1`] = `
<section
  className="NotFoundPage"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
